### DISCUSSION TITLE
Bladeless Tower Fan

### DISCUSSION THREAD

**Discussion by Delgrey** [tuya/tuya-home-assistant#236] [General]
My bladeless tower fan can turn on and off! Woohoo. It's fast too. But... Can I make it do all the other features? Speed? Oscillate? Sleep timer? etc. I got these going with a bit of messing around in Local Tuya before.

[https://www.kogan.com/au/buy/kogan-smarterhome-bladeless-dc-motor-slim-smart-fan-black/](https://www.kogan.com/au/buy/kogan-smarterhome-bladeless-dc-motor-slim-smart-fan-black/)

**Comment by iotinkognito** [comment]
Raise as issue rather than discussion,
